torch/quantization/quantize.py

```diff
@@ -240,15 +239,11 @@ def quantize(model, run_fn, run_args, mapping=DEFAULT_MODULE_MAPPING):
         run_fn: a function for evaluating the prepared model, can be a
             function that simply runs the prepared model or a training loop
         run_args: positional arguments for `run_fn`
-
-    Return:
-        A quantized model
     """
     model.eval()
```

Comment:
can we do a deepcopy of model before this line

Reply:
Wait, didn't we decide not to deepcopy?